cr/api.py

```diff
@@ -478,15 +478,15 @@ def api_get_task(self, task_id: int) -> dict:
 
     def api_poll_task(self, task_id: int) -> dict:
         """
-        Blocking function to poll a task every 10 seconds until it completes.
+        Blocking function to poll a task every 10 seconds until it completes or errors out.
 
-        Returns the completed task dict.
+        Returns the completed or errored task dict.
 
         Raises TimeoutError if the task does not complete after 3 minutes.
         """
         for i in range(18):
             d = self.api_get_task(task_id)
-            if d["status"] == "completed":
+            if d["status"] != "queued":
```

Comment:
Are we handling error tasks upstream then and showing some kind of message to the user?

Reply:
Yes, in cli.py